### DISCUSSION TITLE
Para lograr el logro 'Galaxy Brain', toma nota de lo siguiente

### DISCUSSION THREAD

**Discussion by johnpablo7** [community/community#69361] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

¡Espero que esta respuesta te haya ayudado, y todo lo mejor para conseguir el logro! :)

**Comment by johnpablo7** [comment]
Responda al menos 2 discusiones. Toma nota, NO PUEDES marcar tus propias respuestas a tus propias preguntas. Tienen que ser marcados por otro usuario. Las discusiones deben estar en un repositorio público. Hay un error actual observado por algunos usuarios por el cual marcar sus propias respuestas puede evitar que obtenga la insignia en su lugar.

**Comment by Akash1134** [comment, marked as answer]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
